src/client/refresh/gl3/gl3_main.c

```diff
@@ -684,6 +688,7 @@ GL3_Shutdown(void)
 		GL3_SurfShutdown();
 		GL3_Draw_ShutdownLocal();
 		GL3_ShutdownShaders();
+		GL3_ShutdownLibrary();
```

Comment:
Shouldn't this be done after GL3_ShutdownContext()?
I'm not sure, but it could be that SDL_GL_DestroyContext() uses libgl?

Reply:
Yes good and no relation with gl buffers neither.